tidb-cloud/tidb-cloud-console-auditing.md

```diff
@@ -1,122 +1,170 @@
-| AuditEventExitImportTask | Exit an import task |
-| AuditEventCreateCluster | Create a cluster |
-| AuditEventDeleteCluster | Delete a cluster |
-| AuditEventScaleCluster | Scale a cluster |
-| AuditEventCreateBackup | Make a backup |
-| AuditEventDeleteBackup | Delete a backup |
-| AuditEventRestoreBackup | Restore from a backup |
-| AuditEventUpdateAuditLogStatus | Enable or disable database audit logging |
-| AuditEventCreateAuditLogAccessRecord | Add database audit log filter conditions |
-| AuditEventDeleteAuditLogAccessRecord | Delete database audit log filter conditions |
-| AuditEventUpdateUserRole | Modify user roles |
-
-## Audit log storage policy
-
-- The audit log information is stored in AWS ES.
-- The storage time is 90 days, after which the audit logs will be automatically cleaned up.
-
-## View audit logs
-
-The console audit logs are temporarily accessible only to internal personnel of TiDB Cloud. If you need to view the logs, contact [PingCAP support team](/tidb-cloud/tidb-cloud-support.md).
-
-## Audit log fields
-
-The fields in audit logs include basic fields and extended fields.
-
-The basic fields are as follows:
+To disable console audit logging, take the following steps:
 
-| Field name | Data type | Description |
-|---|---|---|
-| timestamp | timestamp | Time of event |
-| auditEventType | string | Event type |
-| userID | uint64 | User ID |
-| clientIP | string | Client IP |
-| isSuccess | bool | Event result |
+1. In the upper-right corner of the [TiDB Cloud console](https://tidbcloud.com/), click <MDSvgIcon name="icon-top-organization" /> **Organization** > **Console Audit Logging**.
+2. Click **Setting** in the upper-right corner, and then disable console audit logging.
+
+## View console audit logs
+
+You can only view the console audit logs of your organization. 
 
-Extended fields supplement the description information of events based on different event types to ensure the integrity and availability of audit information
+1. In the upper-right corner of the [TiDB Cloud console](https://tidbcloud.com/), click <MDSvgIcon name="icon-top-organization" /> **Organization** > **Console Audit Logging**.
+2. To get a specific part of audit logs, you can filter the event type, operation status, and time range. 
+3. (Optional) To filter more fields, click **Advanced filter**, add more filters, and then click **Apply**. 
+4. Click the row of a log to view its detailed information in the right pane.
+
+## Export console audit logs
+
+To export the console audit logs of your organization, take the following step:
+
+1. In the upper-right corner of the [TiDB Cloud console](https://tidbcloud.com/), click <MDSvgIcon name="icon-top-organization" /> **Organization** > **Console Audit Logging**.
+3. If you need to export a specific part of console audit logs, you can filter through various conditions. Otherwise, skip this step.
+4. Click **Export** and select the desired export format in JSON or CSV. 
```

Comment:
```suggestion
2. (Optional) If you need to export a specific part of console audit logs, you can filter through various conditions. Otherwise, skip this step.
3. Click **Export** and select the desired export format in JSON or CSV. 
```